scalableminds/webknossos
# Create Annotation CompactWrites to speed up Explorative Annotations List

Less lookups in the database for fields that are not used here anyway.

### URL of deployed dev instance (used for testing):
- https://annotationcompactwrites.webknossos.xyz

### Steps to test:
- create some explorative annotations
- should be shown in the list view
- interact with the list view (labels/archive/unarchive/name), refresh, should work

### Issues:
- fixes #3026

------
